.github/workflows/airbyte-ci-tests.yml

```diff
@@ -46,25 +46,13 @@ jobs:
   run-tests:
     needs: changes
     # We only run the Internal Poetry packages CI job if there are changes to the packages on a non-forked PR
-    if: needs.changes.outputs.internal_poetry_packages == 'true'
+    if: needs.changes.outputs.internal_poetry_packages == 'true' && github.event.pull_request.head.repo.fork != true
```

Comment:
Both workflow declare a job named `Internal Poetry packages CI`. This is the name of the required check.
This change is to skip the `Internal Poetry packages CI` we have on internal PRs so that it does not fail the check. 
The `Internal Poetry packages CI` job from the `community_ci` workflow will run and emit the right check output.